### ISSUE TITLE
Material You

### ISSUE THREAD

**Issue by wmontwe** [thunderbird/thunderbird-android#8340]
We have implemented support for Material 3 #6617 and we could implement Material You support now.

> Would be very nice to see the app with a more modern feel.
My suggestion is to bring it up to par with Material You design.

**Comment by armishinwn** [comment]
I agree with the opinion, the application would gain a lot if they implemented dynamic colors.

**Comment by Cart1416** [comment]
thunderbird ui is still on old material from android 10

**Comment by dav23r** [comment]
@Cart1416 you are spamming issue tracker causing unnecessary notifications to people subscribed to it for no gain. Apart from thumbs-up-ing under issue description, there's no value in begging for something that way, please refrain.

**Comment by farzanni** [comment]
Hi! I would like to work on Android 12+ dynamic color support for Thunderbird. My proposed scope is Thunderbird-only: use the platform wallpaper palette in the Bolt Compose theme and the remaining legacy XML-themed surfaces, while preserving K-9 Mail and the existing palette on older Android versions. I opened #11288 before realizing the contribution workflow requires issue alignment and assignment first—sorry about that. Does this approach fit the intended scope of this issue, and may I be assigned to work on it? I am happy to adjust the implementation based on maintainer guidance.

**Comment by jbott-tbird** [comment]
We have decided to use the Thunderbird Bolt design system instead of Material You.

**Comment by beedell-roke** [comment]
@jbott-tbird, then, I have filed [`bolt-design-system/issues/3`](https://github.com/thunderbird/bolt-design-system/issues/3#issue-4960133699), to replace this. [`mozilla-mobile/fenix/issues/20778`](https://github.com/mozilla-mobile/fenix/issues/20778#issuecomment-1372575072) was a reference.

**Comment by beedell-roke** [comment]
@thundernotion, what PRs (or commits) was this resolved in? I don't understand how this could be complete without any linked work.
